### DISCUSSION TITLE
InputNumber with increment/decrement buttons inline

### DISCUSSION THREAD

**Discussion by SteveDowsett** [mantinedev/mantine#6263] [Q&A]
Hey all, i wonder if anyone could help me?

Im trying to produce this form UI and wondered how i could do it?
Firstly, is it possible to have the form input label/description to the left of the actual input like in the image?
and secondly, ive tried to use the InputNumber component, with buttons for the -/+ increments passed as left/right section components, but although the buttons are rendered before and after the actual input text, the text is left aligned so doesnt look right? Would this be a case of rolling my own component using the Input base?

![image](https://github.com/mantinedev/mantine/assets/22001355/eecee1ca-3429-4043-8bed-51ca468d73c7)

**Comment by rtivital** [comment]
1. https://mantine.dev/core/number-input/#incrementdecrement-controls
2. https://mantine.dev/core/number-input/#custom-increment-and-decrement-controls

**Reply by SteveDowsett** [reply]
Thanks, but how can i have the label/description to the left of the actual input like in my image? all the inputs in examples seem to render under the labels/description
